components/View/FormattedDate/index.js

```diff
@@ -9,7 +9,7 @@ const HackedFormattedDate = ({ value, ...otherProps }) => {
         return (
             <FormattedDate
                 {...otherProps}
-                date={value}
+                value={value}
```

Comment:
This beats the whole purpose of this Component.
It's only job was to map between value and date props